osgp/protocol-adapter-dlms/osgp-dlms/src/main/java/org/opensmartgridplatform/dlms/objectconfig/DlmsProfileValidator.java

```diff
@@ -68,7 +74,7 @@ private static void allPqRequestsShouldMatchType(
             .collect(Collectors.joining(", "));
 
     if (!validationError.isEmpty()) {
-      validationErrors.add(createErrorMessage(dlmsProfile, validationError));
+      validationErrors.add(createErrorMessage(dlmsProfile, PQ_ERROR, validationError));
```

Comment:
Waarom is dit een PQ_ERROR?

Reply:
Because this is the error message for a validation for the PQ profiles.